src/compile/data/bin.ts

```diff
@@ -15,7 +15,7 @@ export namespace bin {
     return `format('${format}', ${expr})`;
   }
 
-  function parse(model: Model): Dict<VgTransform[]> {
+  export function parseUnit(model: Model): Dict<VgTransform[]> {
```

Comment:
Thx for explanation-- i kinda prefer the verbose version except that It was the other way round. ( export const parseUnit = parse)

(Replying from mobile)